.github/workflows/test-job.yaml

```diff
@@ -320,6 +320,26 @@ jobs:
         with:
           ray_version: 2.1.0
 
+  test-compatibility-2_2_0:
+    needs:
+      - build_operator
+      - build_apiserver
+      - lint
+    runs-on: ubuntu-latest
+    name: Compatibility Test - 2.2.0
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          # When checking out the repository that
+          # triggered a workflow, this defaults to the reference or SHA for that event.
+          # Default value should work for both pull_request and merge(push) event.
+          ref: ${{github.event.pull_request.head.sha}}
+
+      - uses: ./.github/workflows/actions/compatibility
+        with:
+          ray_version: 2.2.0
```

Comment:
@kevin85421 @architkulkarni @sihanwang41 @wilsonwang371 @Jeffwan 
I recommend testing compatibility of the last three 2.x.x releases _and_ the last 1.x.x.
So I've added 2.2 without removing 1.13.

Reply:
I recommended noting the need to update this compatibility matrix in the Anyscale-internal Ray release instructions.

Reply:
Sounds good, I've added the note in the instructions and linked to this thread.